scripts/devtools/tpl-manager.py

```diff
@@ -150,12 +137,30 @@ def build_deps(args):
 
 
   # Setup spack w/ Uberenv and the appropriate external package/compiler configs.
-  if sexe("python3 {0} --setup-only {1} {2} {3} {4} {5}".format(uberenv_path, prefix_opt, uberenv_project_json_opt, spack_config_dir_opt, spack_upstream_opt, uberenv_spack_url_opt), echo=True): sys.exit(1)
+  sexe("python3 {0} --setup-only {1} {2} {3} {4} {5}".format(uberenv_path, prefix_opt, uberenv_project_json_opt, spack_config_dir_opt, spack_upstream_opt, uberenv_spack_url_opt))
```

Comment:
Again, what if this fails?

Reply:
The `check=True` should cause the tpl manager to fail. Unfortunately, if something fails during a spack command, the current and previous methods failed to cause the tpl manager to quit running. But if uberenv exits with a fail flag, it should stop properly.

Reply:
I'm more concerned with ubuerenv failing to find a spack repository on an air-gapped system where spack needs to be local